rfcs/20230927-construct-tensor-without-user-handle/README.md

```diff
@@ -0,0 +1,103 @@
+# RFC: construct tensor without user handle
+
+## Introduction & Motivation
+
+In benchdnn and unit tests, users often need to allocate memory themselves. 
+Incorrect allocation methods may cause some unexpected errors, such as memory 
+release errors caused by `test::vector`. In fact, the size of the memory can be 
```

Comment:
Do you want to elaborate what is memory release errors caused by `test::vector` here?